docs/source/whatsnew_1_4.md

```diff
@@ -0,0 +1,60 @@
+# What's new in 1.4 🎉🎉
+
+- MAISI: state-of-the-art 3D Latent Diffusion Model
+- VISTA3D: interactive foundation model for segmenting and anotating human anatomies
+- VISTA2D: cell segmentation pipeline
+- Integrating MONAI Generative into MONAI core
+- Geometric Data Support
+
+
+## MAISI: state-of-the-art 3D Latent Diffusion Model
+
+![maisi](../images/maisi_train.png)
+
+NVIDIA MAISI (Medical AI for Synthetic Imaging) is a state-of-the-art three-dimensional (3D) Latent Diffusion Model designed for generating high-quality synthetic CT images with or without anatomical annotations. This AI model excels in data augmentation and creating realistic medical imaging data to supplement limited datasets due to privacy concerns or rare conditions. It can also significantly enhance the performance of other medical imaging AI models by generating diverse and realistic training data.
+
+A tutorial for generating large CT images accompanied by corresponding segmentation masks using MAISI is provided within
+[`project-monai/tutorials`](https://github.com/Project-MONAI/tutorials/blob/main/generation/maisi).
+And it contains the following features:
+- A Foundation Variational Auto-Encoder (VAE) model for latent feature compression that works for both CT and MRI with flexible volume size and voxel size
```

Comment:
I think the word `Foundation` doesn't need to be capitalized.

Reply:
Updated, thanks.